internal/pkg/service/cli/cmd/local/fix_paths.go

```diff
@@ -39,7 +40,8 @@ func FixPathsCommand(p dependencies.Provider) *cobra.Command {
 		},
 	}
 
-	// Flags
-	cmd.Flags().Bool("dry-run", false, "print what needs to be done")
+	fixPathsFlag := FixPathsFlag{}
+	_ = cliconfig.GenerateFlags(fixPathsFlag, cmd.Flags())
+
```

Comment:
```
cliconfig.MustGenerateFlags(FixPathsFlag{}, cmd.Flags())
```

Reply:
```
cliconfig.MustGenerateFlags(FixPathsFlag{
   Foo: Bar
}, cmd.Flags())
```

Reply:
RunE:

```
flags := DefaultFixPathsFlag()
cliconfig.Bind(flags, ....)
```

Reply:
File structure?
```
type FooFlags...

func DefaultFooFlags

func FooCommand
```

Reply:
`flags := DefaultFixPathsFlag()
cliconfig.Bind(flags, ....)`

it will be in the next task